sml_builder/templates/method.html

```diff
@@ -14,6 +14,11 @@
   ]
 } -%}
 {% from "components/status/_macro.njk" import onsStatus %}
+
+{%- block head -%}
+    <link rel="stylesheet" href="{{ url_for('static',filename='css/sml.css') }}" media="all" type="text/css">
```

Comment:
I think you forgot to remove the importing of the sml.css file here

Reply:
Yes you're right